From 3a33d5ccc477e147b5bf33a804e46cd9ebc295eb Mon Sep 17 00:00:00 2001
From: Alex Kontos <MrAlex94@users.noreply.github.com>
Date: Wed, 5 Oct 2022 14:53:01 +0100
Subject: [PATCH] [WFX-200] Disable DoH automatic roll-out

---
 waterfox/browser/app/profile/000-waterfox.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/waterfox/browser/app/profile/000-waterfox.js b/waterfox/browser/app/profile/000-waterfox.js
index 117c2ebfccc9..363b53decf1e 100644
--- a/waterfox/browser/app/profile/000-waterfox.js
+++ b/waterfox/browser/app/profile/000-waterfox.js
@@ -81,6 +81,7 @@ pref("datareporting.healthreport.uploadEnabled", false, locked);
 pref("datareporting.policy.dataSubmissionEnabled", false, locked);
 pref("devtools.debugger.chrome-debugging-host", "127.0.0.1");
 pref("devtools.webide.autoinstallADBExtension", false);
+pref("doh-rollout.disable-heuristics", true); // Make sure DoH doesn't get enabled automatically
 pref("dom.security.unexpected_system_load_telemetry_enabled", false, locked);
 pref("extensions.allowPrivateBrowsingByDefault", true);
 pref("extensions.experiments.enabled", true);
